src/parpeoptimization/optimizationProblem.cpp

```diff
@@ -54,6 +54,87 @@ void *getLocalOptimumThreadWrapper(void *optimizationProblemVp) {
     return result;
 }
 
+void optimizationProblemGradientCheckMultiEps(OptimizationProblem *problem,
+                                      int numParameterIndicesToCheck
```

Comment:
Remove unused argument.

Reply:
which one do you mean is unused?

Reply:
problem is used in 65 and numParameterIndicesToCheck is used in 67.

Reply:
The latter.

> numParameterIndicesToCheck is used in 67.

the name, yes; the value, no.

Reply:
It is assigned to, but never read. Okay, it's "used" but should be removed.

Reply:
https://github.com/ICB-DCM/parPE/pull/315/files/d4a1558eb699a4ee9732a8c51346980dd11d2544#r578287302

Reply:
fixed